frontend/www/js/omegaup/grader/ephemeral.js

```diff
@@ -174,6 +174,7 @@ Vue.use(Vuex);
 let store = new Vuex.Store({
   state: {
     alias: null,
+    hideSubmitButton: false,
```

Comment:
se puede evitar la lógica negativa?

```suggestion
    showSubmitButton: false,
```

(quizás debería ser `false` por default para que no se muestre a menos que se configure para que sí se muestre).